packages/components/src/Autocomplete/Autocomplete.tsx

```diff
@@ -112,15 +118,29 @@ const AutocompleteInner = <
   >,
   ref?: ForwardedRef<unknown>,
 ) => {
-  const { inputValue, options, loading } = restProps;
-  const isInputValueNotEmpty =
-    inputValue !== undefined && inputValue.length >= 1;
-  const isOptionsAvailable = options.length > 0;
-  const isPopperVisible = isInputValueNotEmpty || isOptionsAvailable || loading;
+  const { options, loading, onInputChange } = restProps;
+
+  const [innerValue, setInnerValue] = useState('');
 
+  const isInnerInputValueNotEmpty = innerValue.length >= 1;
+  const isOptionsAvailable = options.length > 0;
+  const isPopperVisible =
+    isInnerInputValueNotEmpty || isOptionsAvailable || loading;
   const isEmpty = checkIsInputEmpty(restProps.value);
   const disableClearable = isEmpty || Boolean(externalDisableClearable);
 
+  const handleInputChange = (
+    event: SyntheticEvent,
+    value: string,
+    reason: AutocompleteInputChangeReason,
+  ) => {
+    if (onInputChange) {
+      onInputChange(event, value, reason);
```

Comment:
```suggestion
      onInputChange?.(event, value, reason);
```

Такой записи не хватит?